src/lib.rs

```diff
@@ -265,8 +267,17 @@ impl<'a> Executor<'a> {
     fn schedule(&self) -> impl Fn(Runnable) + Send + Sync + 'static {
         let state = self.state().clone();
 
-        // TODO: If possible, push into the current local queue and notify the ticker.
-        move |runnable| {
+        move |mut runnable| {
+            //  If possible, push into the current local queue and notify the ticker.
+            let local_queue = state.local_queues.get();
+            if let Some(queue) = local_queue {
+                runnable = if let Err(err) = queue.push(runnable) {
+                    err.into_inner()
+                } else {
+                    state.notify();
```

Comment:
I'm only a bit familiar with how notification is supposed to work in the executor, so this may be incorrect, or we may need to store the waker along with the queue, like in #37.

Reply:
This could wake up a thread other than the one that's being pushed to, which would cause the task to be stuck inside the queue. So the waker itself needs to be kept inside of the `LocalQueue`.

I would recommend [`AtomicWaker`](https://docs.rs/atomic-waker/latest/atomic_waker/struct.AtomicWaker.html) for this purpose, as the item inside of the `ThreadLocal` needs to be `Sync`.

Reply:
Done!